projects/ngx-drag-to-select/src/lib/operators.ts

```diff
@@ -1,14 +1,16 @@
-import { Observable } from 'rxjs/Observable';
+import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
-import { SelectBox, SelectBoxInput, SelectContainerHost } from './models';
+import { SelectBox, SelectBoxInput, SelectContainerHost, MousePosition } from './models';
 import { getRelativeMousePosition } from './utils';
 
 export const createSelectBox = (container: SelectContainerHost) => (
   source: Observable<SelectBoxInput>
 ): Observable<SelectBox<number>> =>
   source.pipe(
     map(([event, opacity, { x, y }]) => {
-      const mousePosition = getRelativeMousePosition(event, container);
+      // explicit cast to MousePosition in order to "use" `MousePosition`
+      // we have to import `MousePosition` otherwise we'll get a build error
+      const mousePosition: MousePosition = getRelativeMousePosition(event, container);
```

Comment:
Do you got a better solution? 
The build was throwing an error because it didn't know `MousePosition`